CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 CHANGELOG
 =========
 
+1.18.4.post1
```

Comment:
the "dev" part of the label means we haven't released it yet, so you can add this in the entry below. Also maybe add an entry for the last PR since we forgot?